projects/fizzbuzz/java/FizzBuzz.java

```diff
@@ -0,0 +1,30 @@
+package Project2;
+import java.util.Scanner;
+
+public class FizzBuzz {
+
+	public static void main(String[] args) {
+		Scanner input = new Scanner(System.in);
+		System.out.println("Enter an integer greater that 1");
+		int in = Integer.parseInt(input.next());
+		input.close();
+		for (int i= 0; i < in; i++) {
+			if(i % 3 != 0 && i % 5!=0) {

```

Comment:
Not a big deal, but inconsistent uses of spaces in formatting of logical operations here and in line 16.